compiler/rustc_trait_selection/src/traits/query/type_op/implied_outlives_bounds.rs

```diff
@@ -3,7 +3,7 @@ use crate::traits::query::outlives_bounds::OutlivesBound;
 use crate::traits::query::Fallible;
 use rustc_middle::ty::{ParamEnvAnd, Ty, TyCtxt};
 
-#[derive(Clone, Debug, HashStable, TypeFoldable, Lift)]
+#[derive(Copy, Clone, Debug, HashStable, TypeFoldable, Lift)]
```

Comment:
Same as previous comment, but with even less of a concern.

Reply:
This one slightly more than the previous one: to easily return the canonicalized query from type ops, the [`QueryTypeOps` are now `Copy`](https://github.com/rust-lang/rust/commit/a519010f05fcc2de990d97d62dd8db3ef873ebfd#diff-9daa09c4315618b964ed8617b325631124c62d4d1479f596f66458dcc2b466caR57) and these `ImpliedOutlivesBounds` [implement that](https://github.com/rust-lang/rust/blob/a519010f05fcc2de990d97d62dd8db3ef873ebfd/compiler/rustc_trait_selection/src/traits/query/type_op/implied_outlives_bounds.rs#L11)